cpp/dspace/doi.cc

```diff
@@ -0,0 +1,100 @@
+#include <iostream>
+#include <regex>
+#include <set>
+#include <unordered_map>
+#include <cstdlib>
+#include "DbConnection.h"
+#include "DbResultSet.h"
+#include "DbRow.h"
+#include "Downloader.h"
+#include "EmailSender.h"
+#include "HttpHeader.h"
+#include "JSON.h"
+#include "StlHelpers.h"
+#include "StringUtil.h"
+#include "VuFind.h"
+#include "util.h"
+
+using namespace std;
+
+const std::string DSPACE_SERVER_URL("https://publikationen.uni-tuebingen.de/rest/items/");
+const std::string DOI_KEY("dc.relation.uri");
+const std::string PUBLIKATION_TITLE_KEY("dc.title");
+unsigned int INSTANCE_COUNT(15000);
```

Comment:
INSTANCE_COUNT doesn't seem to be a proper name... It will be passed to the Downloader to override the time_limit parameter. So a better name could be e.g. DSPACE_TIME_LIMIT.

But the real question here is =>  the default time_limit is 20000, so why do we even bother to explicitly set one to 15000? Maybe we can just leave the parameter empty so the default is taken?